.github/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ### New features
 
+* Fixes bug in `strawberryfields.decompositions.rectangular_symmetric` so its
+  returned phases are all in the interval [0, 2*pi[, and corrects the
```

Comment:
```suggestion
  returned phases are all in the interval `[0, 2*pi]`, and corrects the
```

Reply:
Here I meant to say that the interval is open at the 2pi-end.. is there an unambiguous way to write that?

Reply:
Oh right! So,
```suggestion
  returned phases are all in the interval [0, 2*pi), and corrects the
```